api/engine.py

```diff
@@ -363,14 +452,66 @@ def __get_logs():
     Returns:
         an array with JSON events
     """
+    session = create_connection(__language())
     __api_key_check(app, flask_request, __language())
     try:
         host = __get_value(flask_request, "host")
     except Exception:
         host = ""
-    return jsonify(__logs_to_report_json(host, __language())), 200
+    data = __logs_to_report_json(host, __language())
+    json_object = json.dumps(data)
+    host_temp = session.query(HostsLog).filter(HostsLog.host==host).all()
+    dateFromDB = host_temp[0].date
+    dateFormat=datetime.strptime(dateFromDB, "%Y-%m-%d %H:%M:%S")
+    dateFormat = str(dateFormat).replace("-","_").replace(":","_").replace(" ", "_")
```

Comment:
@aman566 here the db query is not necessary - please revert to your previous code using datetime.now